src/SprykerEco/Zed/Econda/EcondaDependencyProvider.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Econda;
+
+use Spryker\Zed\Kernel\AbstractBundleDependencyProvider;
+use Spryker\Zed\Kernel\Container;
+use SprykerEco\Zed\Econda\Communication\Plugin\CategoryPlugin;
+use SprykerEco\Zed\Econda\Communication\Plugin\ProductsPlugin;
+use SprykerEco\Zed\Econda\Dependency\Facade\EcondaToLocaleFacadeBridge;
+use SprykerEco\Zed\Econda\Dependency\Facade\EcondaToPriceProductFacadeBridge;
+use SprykerEco\Zed\Econda\Dependency\Facade\EcondaToPropelFacadeBridge;
+
+class EcondaDependencyProvider extends AbstractBundleDependencyProvider
+{
+    protected const FACADE_LOCALE = 'FACADE_LOCALE';
+    protected const FACADE_PROPEL = 'FACADE_PROPEL';
```

Comment:
`protected` => `public` : currently the factory throws fatal error since it can't reach this constant. I would suggest a QA on the code.